guides/v2.1/coding-standards/technical-vision/webapi.md

```diff
@@ -88,13 +88,27 @@ Any new design related to Web API must satisfy the following constraints to keep
 
 **GraphQL**
 
-1. Unlimited nesting should be supported during requests for related entities. (For example, get Order => Order Items => Products => Related Products)
-1. Field filtration must be performed with SQL queries. Do not filter on the application layer after you've fetched all possible fields.
+1. Unlimited nesting should be supported during requests for related entities.
+(For example, get Order => Order Items => Products => Related Products) 
+1. Field filtration MUST be performed by the corresponding level. 
```

Comment:
"by the corresponding level" - it's not clear how to use this information. What is "corresponding level"?